mapping/sustainability.yaml

```
@@ -550,7 +550,7 @@
     module: Climate finance
     indicator: Policy coherence
     disclosure format: Select from a list of international commitments and clarify how the project is aligned with it \[codelist, free text to explain\].
-    mapping: See mapping for data point "Policy coherence" in Institutional extension adding the selected codes to the `policyAlighment.policies` array.
+    mapping: See mapping for data point "Policy coherence" in Institutional extension adding the selected codes to the `policyAlignment.policies` array.
```

Please remove the example for this mapping so as not to have to maintain it in multiple places.

This change will cause a build error until #428 is merged and 0.9-dev is merged into this branch.

```suggestion
    mapping: See [policy coherence (institutional module)](institutional-policy-coherence).
```

---

Unresolved this just to keep the note that the tests are failing until #428 is merged